dali/python/nvidia/dali/types.py

```diff
@@ -55,15 +55,17 @@ def _not_implemented(val):
         DALIDataType.INTERP_TYPE : ("nvidia.dali.types.DALIInterpType", lambda x: DALIInterpType(int(x))),
         DALIDataType.TENSOR_LAYOUT : (":ref:`layout str<layout_str_doc>`", lambda x: str(x)),
         DALIDataType.PYTHON_OBJECT : ("object", lambda x: x),
-        DALIDataType._TENSOR_LAYOUT_VEC : (":ref:`layout str<layout_str_doc>`", _to_list(lambda x: str(x)))
+        DALIDataType._TENSOR_LAYOUT_VEC : (":ref:`layout str<layout_str_doc>`", _to_list(lambda x: str(x))),
+        DALIDataType._DALI_DATA_TYPE_VEC : ("DALIDataType", _to_list(lambda x: str(DALIDataType.DATA_TYPE)))
```

Comment:
```suggestion
        DALIDataType._DALI_DATA_TYPE_VEC : ("nvidia.dali.types.DALIDataType", _to_list(lambda x: str(DALIDataType.DATA_TYPE)))
```
To match what is done in line 55